lopper/assists/openamp_xlnx.py

```diff
@@ -743,6 +743,9 @@ def xlnx_remoteproc_construct_cluster(tree, cpu_config, elfload_nodes, new_ddr_n
         for carveout in elfload_nodes:
             if carveout.props("status") != []:
                 srams.append(carveout.phandle)
+                # FIXME for each sram, add 'power-domain' prop for kernel driver
+                carveout + LopperProp(name="power-domain",
+                                      value=copy.deepcopy( carveout.props("power-domains")[0].value ))
```

Comment:
Note: if you want the entire property to be deep copied for a new one,  LopperProp (and LopperNode) implement the __call__ magic function (and do a deepcopy by default). So you can do new_prop = old_prop()

But this is ok, as is .. that is just for reference.